backend/libraries/error_codes/src/lib.rs

```diff
@@ -181,6 +181,7 @@ pub enum OCErrorCode {
     IdentityLinkRequestNotFound = 337,
     InvalidBotActionScope = 338,
     WebhookNotFound = 339,
+    WebhookInvalid = 340,
```

Comment:
All of the "X is invalid" error code names start with `Invalid...` so that they're easy to find (via intellisense).